src/ign-frontend/src/views/ImagIA.vue

```diff
@@ -4,9 +4,6 @@
         h1="Image Go AI 🤖"
         h2="Use the power of the PaletteNet to recolor your images"
       />
-    <center>
-      <a class="github-button" href="https://github.com/Schrodinger-Hat/ImageGoNord" data-color-scheme="no-preference: light; light: light; dark: dark;" data-size="large" data-show-count="true" aria-label="Star Schrodinger-Hat/ImageGoNord on GitHub">Star</a>
```

Comment:
If you still want the widget at the top (Its at the bottom already) I can take a look why this is'nt working. But imo, is not necessary

Reply:
Let's take it as a new issue: I've added there to give more awareness to this repository as the goal is to get the project known here on GitHub :)